src/conv-core/msgmgr.C

```diff
@@ -168,7 +169,9 @@ CmmTable CmmPup(pup_er p, CmmTable t, CmmPupMessageFn msgpup)
       int ntags, *tags;
       void *msg;
       pup_int(p, &ntags);
-      tags = (int*) malloc(ntags*sizeof(int));
+      CmiEnforce(ntags <= std::numeric_limits<int>::max());
```

Comment:
Would this condition ever not be true? (same for threads.C, pup_stl.h)

Reply:
It looks like LGTM no longer identifies alerts at these locations, so I've removed the numeric limits checks.